sw/host/opentitantool/src/command/image.rs

```diff
@@ -155,7 +155,8 @@ impl CommandDispatch for DigestCommand {
         _context: &dyn Any,
         _transport: &TransportWrapper,
     ) -> Result<Option<Box<dyn Serialize>>> {
-        Ok(None)
+        let image = image::Image::read_from_file(&self.image)?;
+        Ok(Some(Box::new(format!("{}", image.compute_digest()))))
```

Comment:
Lets define a struct above like:
```
#[derive(serde::Serialize)]
struct DigestResponse {
    pub digest: String,
}
```

And then return that struct here:
```
Ok(some(Box::new(DigestResponse {
    digest: format!(...),
})))
```

Lets also add an `bin: Option<PathBuf>` to `DigestCommand` and use that to write the digest out as a binary file:
```
if let Some(bin) = &self.bin {
    write_to_file(bin, image.compute_digest())?;
}
```

I think we'll want the binary representation available for the HSM based signing infrastructure.
WDYT?